### ISSUE TITLE
display format

### ISSUE THREAD

**Issue by callmejm** [wuweijian1997/FlutterCountdownTimer#6]
when use , it show 00:05:00 , how can only 05:00 ?

**Comment by wuweijian1997** [comment]
Can this solve your problem?I will send another version to solve this problem in the next few days.
```
@override
  Widget build(BuildContext context) {
    int endTime = DateTime.now().millisecondsSinceEpoch + 1000 * 60;
    return Scaffold(
      body: Center(
        child: CountdownTimer(endTime: endTime,
        hoursTextStyle: TextStyle(fontSize: 0, color: Colors.transparent),
        hoursSymbol: "",
        )
      ),
    );
  }
```

**Comment by wuweijian1997** [comment]
hello,I released version ^1.3.0.Now you can implement custom widgets through widgetBuilder.If you have other ideas, please feel free to give me feedback

demo code
```

 CountdownTimer(
            endTime: endTime,
            widgetBuilder: (BuildContext context, CurrentRemainingTime time) {
              if(time == null) {
                return Text('Game over');
              }
              List<Widget> list = [];
              if(time.days != null) {
                list.add(Row(
                  children: <Widget>[
                    Icon(Icons.sentiment_dissatisfied),
                    Text(time.days.toString()),
                  ],
                ));
              }
              if(time.hours != null) {
                list.add(Row(
                  children: <Widget>[
                    Icon(Icons.sentiment_satisfied),
                    Text(time.hours.toString()),
                  ],
                ));
              }
              if(time.min != null) {
                list.add(Row(
                  children: <Widget>[
                    Icon(Icons.sentiment_very_dissatisfied),
                    Text(time.min.toString()),
                  ],
                ));
              }
              if(time.sec != null) {
                list.add(Row(
                  children: <Widget>[
                    Icon(Icons.sentiment_very_satisfied),
                    Text(time.sec.toString()),
                  ],
                ));
              }

              return Row(
                mainAxisAlignment: MainAxisAlignment.center,
                children: list,
              );
            },
          ),
```
